airbyte-integrations/connectors/source-bamboo-hr/source_bamboo_hr/utils.py

```diff
@@ -1,17 +0,0 @@
-#
-# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
-#
-
-
-def convert_custom_reports_fields_to_list(custom_reports_fields: str) -> list:
-    return custom_reports_fields.split(",") if custom_reports_fields else []
-
-
-def validate_custom_fields(custom_fields, available_fields):
-    denied_fields = []
-    for custom_field in custom_fields:
-        has_access_to_custom_field = any(available_field.get("name") == custom_field for available_field in available_fields)
-        if not has_access_to_custom_field:
-            denied_fields.append(custom_field)
-
-    return denied_fields
```

Comment:
I think this is an important function used in check connection to make sure the custom report you're building is correct. wdyt?

Reply:
@parthiv11 did you check my comment here? What is your opinion about removing this from the connector?

Reply:
Here it is using **MetaFields** to fetch all available fields and check if user inputted any wrong field, if yes it throws error.

How this can be done in ui builder , any idea?

Reply:
It's not possible. You need to use at least the low-code framework.